qemu/tests/cfg/hotplug_mem_negative.cfg

```diff
@@ -0,0 +1,43 @@
+- hotplug_mem_negative:
+    type = hotplug_mem_negative
+    mem_fixed = 4096
+    slots_mem = 4
+    size_mem = 1G
+    maxmem_mem = 32G
+    guest_numa_nodes = "node0"
+    ppc64,ppc64le:
+        threshold = 0.15
+    policy_mem = default
+    target_mems = "plug1"
+    set_addr = no
+    variants:
+        - overcommit_host_mem:
+            size_mem = 1000G
+            keywords = "Cannot allocate memory"
+        - min_mem_size:
+            size_mem = 0G
+            keywords = "memory-backend-ram.size\' doesn\'t take value \'0\'"
+        - invalid_backend_ram_option:
+            policy_mem = bind
+            host-nodes = 3
+            keywords = "cannot bind memory to host NUMA nodes"
+        - invalid_numa_node:
+            node_dimm_plug1 = 2
+            keywords = "DIMM property node has value 2"
+        - wrong_size:
+            size_mem = 3
+            keywords = "Hotplugged memory size must be a multiple"
```

Comment:
Need update keywords. 
05:07:10 INFO | Context: Error happen plug1: QMP command 'device_add' failed    (arguments: OrderedDict([('driver', 'pc-dimm'), ('id', 'dimm-plug1'), ('memdev', 'mem-plug1')]),    error message: {u'class': u'GenericError', u'desc': u'backend memory size must be multiple of 0x200000'})

Reply:
It seems that the keywords of x86 and ppc are not exactly the same, what do you think of changing to keywords = "memory size must be a multiple"